### ISSUE TITLE
Multiple or custom replace tag

### ISSUE THREAD

**Issue by vnq** [KEINOS/parsedown-extension_table-of-contents#17]
Could we support either multiple replace tags like `[toc]`so we can allow `[toc]`, `[[_TOC_]]`, `{:toc}` etc.

For reference see: https://gitlab.com/gitlab-org/gitlab/-/issues/21901#note_280349199

Or a way to override the `const TAG_TOC` so I can change it to my liking, in my case `[[_TOC_]]`.

**Comment by KEINOS** [comment]
Do you mean a user-defined `ToC` tag? Cool! Let's do it.

By chance, I was refactoring some in [organize-test-suite](https://github.com/KEINOS/parsedown-extension_table-of-contents/tree/organize-test-suite) branch and the below part might help us implement this. A `getter` method to get the `ToC` tag.

https://github.com/KEINOS/parsedown-extension_table-of-contents/blob/be69f9a4a5bdee465805910b0c117d95d99a782f/Extension.php#L45-L53

https://github.com/KEINOS/parsedown-extension_table-of-contents/blob/be69f9a4a5bdee465805910b0c117d95d99a782f/Extension.php#L206-L218

https://github.com/KEINOS/parsedown-extension_table-of-contents/blob/be69f9a4a5bdee465805910b0c117d95d99a782f/Extension.php#L220-L236

So, implementing the `setter` method might satisfy our needs. Doesn't it?

As soon as I merge the `organize-test-suite` branch, I will draft a PR for this request!

**Comment by KEINOS** [comment]
@vnq 

I believe there must be a smarter way to implement but I released [v1.1.2](https://github.com/KEINOS/parsedown-extension_table-of-contents/releases/tag/v1.1.2) which implements your request. Thank you for the cool request!

Please update the plug-in and use the `setTagToc('[[_TOC_]]')` method to set the ToC markdown tag you like. Hope this suits your needs! 🤞

```php
<?php
require_once __DIR__ . '/vendor/autoload.php';

$text_markdown = file_get_contents('SAMPLE.md');

$Parsedown = new \ParsedownToC();
$Parsedown->setTagToc('[[_TOC_]]');

$html = $Parsedown->text($text_markdown);
echo $html . PHP_EOL;
```

These are the tests for the custom ToC tag

- [test_vanilla_user_defined_toc_tag_style1.sh](https://github.com/KEINOS/parsedown-extension_table-of-contents/blob/master/tests/test_vanilla_user_defined_toc_tag_style1.sh) -> for `"[[TOC]]"`
- [test_vanilla_user_defined_toc_tag_style2.sh](https://github.com/KEINOS/parsedown-extension_table-of-contents/blob/master/tests/test_vanilla_user_defined_toc_tag_style2.sh) -> for `"{:toc}"`
- [test_vanilla_user_defined_toc_tag_style3.sh](https://github.com/KEINOS/parsedown-extension_table-of-contents/blob/master/tests/test_vanilla_user_defined_toc_tag_style3.sh) -> for `"[[_TOC_]]"`

**Comment by vnq** [comment]
@KEINOS great job thanks for this!
